### DISCUSSION TITLE
Feature Request: Move Window and Goto to Next/Previous Virtual Desktop

### DISCUSSION THREAD

**Discussion by airtonix** [FancyWM/fancywm#267] [Q&A]
I'm looking for same functionality that I have enjoyed in Linux for the last 20 years: 

- in a single press: <my chosen keyboard short cut A>
  - move to previous virtual desktop and take current window with me
- in a single press: <my chosen keyboard short cut B>
  - move to next virtual desktop and take current window with me

Currently, FancyWM only supports a quarter of the above requirements: 

![image](https://github.com/FancyWM/fancywm/assets/61225/69832a6c-1eef-4797-baab-932a2874ad23)

- disable activation hotkey for `Move window to previous desktop`
- set `Move window to previous desktop` to <my chosen keyboard short cut A>

I notice that the Keyboard Shortcut Settings above and https://github.com/FancyWM/fancywm/wiki/Scripting shows only : 

> - SwitchToPreviousDesktop - switch to the previous virtual desktop (if any) [v2.7.0+]
> - SwitchToDesktopN - switch to the specified virtual desktop
> - MoveToPreviousDesktop - move the focused window to the previous virtual desktop (if any) [v2.7.0+]
> - MoveToDesktopN - move the focused window to the specified virtual desktop

So there's a missing: 

> - MoveToNextDesktop - move the focused window to the previous virtual desktop (if any)

I'm not really looking to use the Chord style of interaction, I'm not a Vim or Emac user so this doesn't appeal to me. I just want to press a set of keys in one go and have the action carried out.

So what's required is: 

- One more keyboard shortcut for `Move window to next desktop`
- a tickbox `Follow focus` under both `Move window to previous desktop` and `Move window to next desktop`

**Comment by kfyangc** [comment]
Not implemented yet.

However, I find a method to achieve this, which requires using `AutoHotkey`, and here is the code:

```
#Requires AutoHotkey v2.0
F11::{
    active_id := WinGetID("A")
    WinHide active_id
    Send "^#{Right}"
    WinShow active_id
    WinActivate active_id
}
F10::{
    active_id := WinGetID("A")
    WinHide active_id
    Send "^#{Left}"
    WinShow active_id
    WinActivate active_id
}
```

I just copy the code from [here](https://blog.csdn.net/qq_45354906/article/details/136170216).
